cpu/kinetis_common/periph/pwm.c

```diff
@@ -35,6 +35,22 @@ static inline FTM_Type *ftm(pwm_t pwm)
     return pwm_config[pwm].ftm;
 }
 
+static void poweron(pwm_t pwm)
```

Comment:
Actually, I personally hate these `_xx` naming schemes, as I think it is no redundant after all: if it would be a public function, it would start with the module name prefix (e.g. `pwm_`). As the function does not start with this prefix -> it is internal...